README.md

```diff
@@ -43,23 +43,24 @@ The fastest way to get results, is to:
+    3. C) This setting determines how many looks the app will keep from the old generation when generating a new generation. Only looks with the highest rating you selected will be kept (so if the highest rating you used selected was '4', then it will keep this many looks of rating '4'). Looks with rating 1 will never be kept. If you want all looks with the highest rating you selected to be kept, then you can select 20 here.
+    4. D) If you want to be able to automatically scan subdirectories of your appearance folder for appearances as well, set this to Yes.
+    5. E) If you need more space, you can choose Yes to get a smaller rating window. This smaller window size should fit a 720p resolution monitor.
 
-8.	Press "Initialize Population". If you use Gaussian Samples, this will take a while (~1 minute).
-9.	Rate Children. You now see Generation 1 of your 20 children. Each child is a new generated appearance based on your previous choices. I made a "VAM Evolutionary Character Creation Companion" save which you can open in VAM. This save helps with the rating process. When you open this save in VAM, you can click on one of the 20 numbers to the left, to see each of the generated appearances by the python app. And then rate them in the python app. I have both VAM and the python app open with the windows next to eachother. The higher you rate each child, the bigger the chance that it will be used to create children for the next generation. The highest rated appearance (or in the case of a tie, the first child with that rating), will always be kept and stored as Child 1 for the new generation.
-10.	Generate Next Population. If you're done with rating, you can press the Generate Next Population. 
-11.	Return to Step 9. You can keep doing this until you are happy with the results.
-12.	When you are happy with one of the generated children, make sure to save that appearance to a custom name. If you rerun the app, all cihldren appearances will be overwritten, so better save it to a new one.
+9. Press "Initialize Population". If you use Gaussian Samples, this will take a while (~1 minute).
+10. Rate Children. You now see Generation 1 of your 20 children. Each child is a newly generated appearance based on your previous choices. I made a "VAM Evolutionary Character Creation Companion" save which you can open in VAM. This save helps with the rating process. When you open this save in VAM, you can click on one of the 20 numbers to the left, to see each of the generated appearances by the python app. And then rate them in the python app. I have both VAM and the python app open with the windows next to each other. The higher you rate each child, the bigger the chance that it will be used to create children for the next generation. The appearances you rated '5' will always be kept as the first appearance(s) of the next generation.
```

Comment:
This should probably read like: "By default, the highest rated appearance, called the 'elite', (or in the case of a tie, the first child with that rating), will always be kept and stored as Child 1 for the new generation. You can increase the amount of elites to be automatically saved for the next generation in the options, by changing "Max kept elites (highest rated):" to a number higher than 1. One word of caution though: if you keep too many elites, this will hurt the diversity of your gene pool. In simple words: keep this number low (below 3 or so).

Reply:
done